oshi-core/src/main/java/oshi/hardware/platform/unix/freebsd/FreeBsdDisks.java

```diff
@@ -66,6 +66,7 @@ public static boolean updateDiskStats(HWDiskStore diskStore) {
             // In KB
             diskStore.setReadBytes((long) (ParseUtil.parseDoubleOrDefault(split[3], 0d) * 1024));
             diskStore.setWriteBytes((long) (ParseUtil.parseDoubleOrDefault(split[4], 0d) * 1024));
+            diskStore.setQueueLength((long) (ParseUtil.parseDoubleOrDefault(split[5], 0d)));
```

Comment:
Nitpick. For documentation you stuck these under the "In KB" header but aren't multiplying by 1024.  So leave a space or add another comment to indicate the units of this.

Also I'm wondering why we parse double and cast to long (same thing in some previous lines that you didn't change) rather than parsing directly to long...

Reply:
Not sure how to validate if we can parse to long directly ...

Reply:
Dug into the source code. Here's the format headers for FreeBSD:
```
"device     r/s   w/s    kr/s    kw/s qlen svc_t  %%b  "
"%-8.8s %5.1Lf %5.1Lf %7.1Lf %7.1Lf %4" PRIu64 " %5.1Lf %3.0Lf "
```
So all the outputs are floating point except:
- qlen is `%4" PRIu64 "` which is an unsigned 64 bit long...
- %b (busy) is truncated to 0 decimals so could be parsed as an int